### PR TITLE
password entry is foreground window when DB locked

### PR DESCRIPTION
**Pull request by AshleyT3**
When pwsafe DB is locked while pwsafe is in non-systray "taskbar" mode, the password entry dialog becomes the pwsafe app window. This allows pwsafe DB unlock on activation to present the password entry dialog as the foreground window with focus.
https://github.com/pwsafe/pwsafe/issues/1014

### CODE REVIEW

File: src/ui/Windows/MainView.cpp [hunk 1]
```diff
@@ -2558,7 +2558,11 @@ void DboxMain::OnTimer(UINT_PTR nIDEvent)
     if (prefs->GetPref(PWSprefs::UseSystemTray)) {
```
Comment (ronys) [inline src/ui/Windows/MainView.cpp:2558]: I've negated the predicate and moved the call to ShowWindow(SW_HIDE) outside of the if statement to make it a bit clearer.

File: src/ui/Windows/PasswordSafe.rc
Comment (ronys) [inline src/ui/Windows/PasswordSafe.rc:1]: No idea why a couple of minor changes made git mark this entire file as changed. Line endings?

Comment (ronys) [reply src/ui/Windows/PasswordSafe.rc:1]: Line endings it is. Fixing.

**Comment by ronys** [comment]
Nice work - thanks!

File: src/ui/Windows/PasskeyEntry.cpp [hunk 2]
```diff
@@ -259,13 +274,6 @@ BOOL CPasskeyEntry::OnInitDialog(void)
     return FALSE;
   }
 
-  // Following works fine for other (non-hotkey) cases:
-  if (m_index == GCP_RESTORE || m_index == GCP_WITHEXIT) {
-    SetWindowPos(&CWnd::wndTopMost, 0, 0, 0, 0, SWP_NOMOVE | SWP_NOSIZE);
-    SetActiveWindow();
-    SetForegroundWindow();
-  }
```
Comment (dskrvk) [inline src/ui/Windows/PasskeyEntry.cpp:267]: This blocks seems to have been removed completely - why?

Comment (AshleyT3) [reply src/ui/Windows/PasskeyEntry.cpp:267]: It was redundant.

**Pull request merged**